src/oop/app/src/Transporters/GuzzleAdapter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace src\oop\app\src\Transporters;
+
+use GuzzleHttp\Client;
+
+class GuzzleAdapter implements TransportInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function getContent(string $url): string
+    {
+        $client = new Client();
```

Comment:
please use dependency injection for Client (inject it into constructor)

Reply:
> please use dependency injection for Client (inject it into constructor)

All modifications done.